web/src/@types/mui-theme.d.ts

```diff
@@ -29,3 +30,14 @@ declare module '@mui/material/Typography' {
     overline: false
   }
 }
+
+declare module '@mui/material/styles' {
+  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+  interface Palette {
+    tertiary: PaletteColor
+  }
+  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+  interface PaletteOptions {
+    tertiary: PaletteColor
+  }
+}
```

Comment:
It changed the colors of the toggle button when while using contrast mode:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/1b244c37-bacd-49b3-bc4d-d8949ee11282" />

Reply:
I asked Toni about the contrast and told me to adjust the contrast so it matches accessibility guidelines of 4.5:1.
She also said that the contrast is fine on one we have at https://future.integreat.app/muenchen/de
here's a screenshots of that:
<img width="400" alt="Bildschirmfoto 2025-07-29 um 16 13 49" src="https://github.com/user-attachments/assets/b03f038d-0c66-434f-8962-0cf1dcfadbc8" />
<img width="400" alt="Bildschirmfoto 2025-07-29 um 16 14 23" src="https://github.com/user-attachments/assets/1d880661-3256-4549-8cfa-972c32999d0a" />

Reply:
True, I also think before had more contrast/looked a little better. However, now its correctly using our primary color 🤔